guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: guide
+title: Getting to know your users
+description: How best to understand and develop knowledge about your users.
+parent: Onboarding
+nav_order: 1
+permalink: /guide/onboarding/getting-to-know-your-users/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/getting-to-know-your-users.svg"
+   alt-text = "Bitcoin, key and lock"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Getting to know your users
+
+When building onboarding experiences, it is important to understand and develop knowledge about your users.
+Within the Bitcoin ecosystem, there is a wide range in the level of expertise. Identifying who you are building for is crucial, as these user groups value and understand varying characteristics of coin management tools. For example, “rookies” often favor convenient, easy-to-use wallets, while well-versed bitcoin owners tend to prioritize security[^1].
+
+## Observe your potential users
+This observation could take the form of user research, user interviews and surveys aimed at a range of participants and observe what they do, how they think, and what they want. Let's consider some things we might want to ask ourselves when improving the onboarding experience for new users:
```

Comment:
```suggestion
This observation could take the form of user research, user interviews and surveys aimed at a range of participants to observe what they do, how they think, and what they want. Let's consider some things you might want to ask yourself when improving the onboarding experience for new users:
```

In the [content guidelines](https://bitcoin.design/guide/contribute/content-guidelines/#speak-to-the-reader), we recommend speaking directly to the reader.